src/components/jobManager.ts

```diff
@@ -228,16 +228,16 @@ export class JobManager {
 						false,
 					]);
 				}
-
-				const otherJobHashes =
-					this.#uriHashJobHashSetManager.getRightHashesByLeftHash(
-						uriHash,
-					);
-
-				for (const jobHash of otherJobHashes) {
-					this.#uriHashJobHashSetManager.delete(uriHash, jobHash);
-					this.#jobMap.delete(jobHash);
-				}
+				// We should not delete the job after "apply". User may want to create PR for this job
```

Comment:
I am okay with removing these statements, but how would these jobs be visible in the UI?

Reply:
I will probably add third type of tree items for accepted jobs, and it will contain only Create PR button